src/language-markdown/whitespace.js

```diff
@@ -144,8 +146,8 @@ function canBeConvertedToSpace(path, value, adjacentNodes) {
   ) {
     return false;
   }
-  const previousLastChar = adjacentNodes.previous.value?.at(-1);
-  const nextFirstChar = adjacentNodes.next.value?.[0];
+  const previousLastChar = adjacentNodes.previous.value.at(-1);
```

Comment:
characterBefore/characterAfter?

Reply:
Are `characterJustBefore`/`characterJustAfter` too long?

Reply:
I think `before` should be fine. If I didn't misunderstood.

Reply:
@fisker  `characterBefore` got shorter but `characterAfter` longer...